data/competencies.yml

```diff
@@ -1,16 +1,293 @@
+  domain: null
+
+- id: 9098cc7f-a50a-48df-b84f-c4ee9fa47dbe
+  summary: Is able to explain why using CI/CD pipelines for automation is beneficial, and uses them for basic functions
+  examples:
+    - Restarts broken build.
+    - Is able to interpret CI output and articulate what is failing.
+    - Can fix simple problems or asks for help.
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: ae32dc05-3e53-47d2-a027-345eb42dffa3
+  summary: Occasionally uses monitoring tools to identify problems
+  examples:
+    - Occasionally uses Pingdom to identify problems
+    - Occasionally uses Grafana to identify problems.
+  supportingUrls: []
+  description: null
+  level: junior
   area: technical
   domain: null
 
+- id: 70ae7051-3aed-42a0-be39-6c8a381eeaaf
+  summary: Writes pull requests that are helpful both to the reviewer and those debugging the the code later
+  examples:
+    - PR descriptions that explain why the PR is needed
+    - Links relevant Jira ticket.
+    - Adds screenshots to show changes if applicable.
+    - Provides context that they learned when researching this PR.
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: 57c50830-f2ef-4c61-97fa-ec80b0624473
+  summary: Regularly reads peers' PRs and asks questions in order to learn.
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: 5d637365-7e38-432d-bcec-8652f06d8109
+  summary: Remembers to write tickets and bug reports to track the work expected of them, and can do so with guidance
+  examples:
+    - Checks there is a corresponding ticket or bug report before starting any work, and flags it if not.
+    - Writes reasonable ""steps to reproduce"" in a bug report when prompted.
+    - Adds relevant links and screenshots to pages affected by a bug when prompted.
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: dcfbf24d-04e2-4029-88a0-93479e4d04a8
+  summary: Receives feedback and acts upon it
+  examples:
+    - Implements requested changes on a pull request
+    - Responds to comments on a Google document
+    - Receives feedback from the team that sometimes they don't seem well prepared for morning stand-ups, so takes 5 minutes before stand-ups to gather thoughts about what the status of their work is
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: c8d62965-dd71-4a90-83df-a4c924a40fd8
+  summary: Is able to recognise when they are having difficulties, provides information so that other people can help
+  examples:
+    - Recognises personal work-related problems such as falling behind, not feeling confident, or feeling overwhelmed
+    - Communicates these problems this to their line manager, mentor or tech lead *immediately* so that help and guidance can be provided.
```

Comment:
[Question] Is the emphasis on immediately necessary here? It seems a little forceful